Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/README.md

```diff
@@ -1054,6 +1054,226 @@ Gets compliance search action from the Security & Compliance Center.
+
+### o365-sc-compliance-case-create
+
+***
+Create eDiscovery cases in the Microsoft Purview compliance portal.
+
+#### Base Command
+
+`o365-sc-compliance-case-create`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| case_name | Case name create. | Required | 
+| case_type | "AdvancedEdiscovery: used to manage legal or other types of investigations.<br/> ComplianceClassifier: This type of case corresponds to a trainable classifier.<br/> DataInvestigation: Data investigation cases are used to investigate data spillage incidents.<br/> DSR: Data Subject Request (DSR) cases are used to manage General Data Protection Regulation (GDPR) DSR investigations.<br/> eDiscovery: eDiscovery (also called eDiscovery Standard) cases are used to manage legal or other types of investigations.<br/> This is the default value.<br/> InsiderRisk: Insider risk cases are used to manage insider risk management cases.<br/> Typically, insider risk management cases are manually created in the Microsoft Purview<br/> compliance portal to further investigate activity based on a risk alert.<br/> SupervisionPolicy: This type of case corresponds to communication compliance policy."<br/>. Possible values are: AdvancedEdiscovery, ComplianceClassifier, DataInvestigation, DSR, eDiscovery, InsiderRisk, SupervisionPolicy. Default is eDiscovery. | Optional | 
+| description | Case description. | Optional | 
+| external_id | Case external Id. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365.SecurityAndCompliance.ComplianceCase.Name | String | Case name. | 
+| O365.SecurityAndCompliance.ComplianceCase.Status | String | Case status. | 
+| O365.SecurityAndCompliance.ComplianceCase.CreatedDateTime | String | Case created date time. | 
+
+### o365-sc-compliance-case-list
+
+***
+List different types of compliance cases in the Microsoft Purview compliance portal
+
+#### Base Command
+
+`o365-sc-compliance-case-list`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| identity | List cases by identity. | Optional | 
+| case_type | List cases by type. Possible values are: AdvancedEdiscovery, ComplianceClassifier, DataInvestigation, DSR, eDiscovery, InsiderRisk, SupervisionPolicy. | Optional | 
+| limit | Limit returned cases list size. Default is 50. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365.SecurityAndCompliance.ComplianceCase.Name | String | Case name. | 
+| O365.SecurityAndCompliance.ComplianceCase.Status | String | Case status. | 
+| O365.SecurityAndCompliance.ComplianceCase.GUID | UUID | Case GUID. | 
+| O365.SecurityAndCompliance.ComplianceCase.CreatedDateTime | String | Case created date time. | 
+
+### o365-sc-compliance-case-delete
+
+***
+Removes compliance cases from the Microsoft Purview compliance portal or the Microsoft Purview compliance portal.
```

Comment:
I think this is clearer. 
Removes the Microsoft Purview compliance portal or compliance cases from the Microsoft Purview compliance portal.

Reply:
I have copied it from the microsoft official doc site.
[](https://learn.microsoft.com/en-us/powershell/module/exchange/remove-compliancecase?view=exchange-ps)
It is unclear.
I will use the following :
_Removes compliance cases from the Microsoft Purview compliance portal._